index.html

```diff
@@ -2,58 +2,68 @@
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
 </head>
-    
+
     <body>
-        <p> hey </p>
-        <img class="icons" src="images/site-icons.png" />
-        
-        <div class="row">
         <header>
-            <a href="index.html"><img src="images/logo.png" alt="Mica Shenk" /></a>
-        </header>
-        
-        <nav class="row">
-                <ul>
-                    <li><a href="index.html"><p>HOME</p></a></li>
-                    <li><a href="portfolio/index.html"><p>PORTFOLIO</p></a></li>
-                </ul>
-        </nav>
-        </div>
-        <section>
-            <div id="about-me" class="row">
-        <img src="images/portrait.jpg"/>
-            <div id="me">
-                <h2>hello!</h2>
-        <p>My name is Mica Shenk. I am twenty years old and am a junior at Cedarville University studying graphic and web design. I love the outdoors and get a lot of inspiration from the beautful world around me. This site contains my resume and some portflio pieces that I have made over the past few years. Thanks for visiting! </p>
-            </div>
+            <a href="index.html"><img class="logo" src="images/logo.png" /></a> 
+            <div class="social-media">
+                <a href="https://www.instagram.com/mica.frey/"><img src="images/instagram.png" /></a>
+                <a href="https://www.facebook.com/micafrey1"><img src="images/facebook.png" /></a>
+                <img src="images/email.png" />
             </div>
-            <div id="section-two" class="row">
-                <div id= "option1">
-                    <a href="docs/shenk-resume.pdf"><h1>R</h1>
-                    <div class="description">
-                    <h2>Resume</h2>
-                    <p>Here is my professional resume. It includes various skills, experiences, and interests I have. </p>
-                        </div>
-                    </a>
+            <div class="main">
+                <img class="mica" src="images/mica.png" />
+                <div class="welcome">
+                    <h1>Hi there.</h1>
+                    <h1>I'm <span><img class="name" src="images/name.png" /></span></h1>
+
+                    <p class="resume"><a href="images/MicaFrey_Resume.pdf">View my resume!</a></p>
                 </div>
-                <div id= "option2">
-                    <a href="portfolio/index.html"><h1>P</h1>
-                    <div class="description">
-                        <h2>Portfolio</h2>
-                    <p> One thing I love about school is that it is an environment that fosters growth, allowing me to explore a lot of different avenues in design. I want to share just a few of the pieces I have done this past year. </p>
-                    
             </div>
-        </a>
-                    </div>
-                </div>
-        </section>
+        </header>
+        <table class="portfolio">
```

Comment:
Micah, we cannot use table approach for layout in today's web world. This must be reworked or you will be laughed at. The My Portoflio heading should be an h2 outside of the list of items. Then if you follow that with a `ul` containing `li` and in each of these a hyperlink wrapped around the image and label that will be a better markup solution. Then style wise, add a class of `portfolio` on the `ul` and set it to:

```scss
.portfolio {
  display: flex;
  flex-flow: row wrap;
  
  li { 
    box-sizing: border-box;
    width: 33%; /* Or whatever width you want these to be */
    padding: 0 10px /* or whatever spacing you want between items */
  } 

  img {
    max-width: 100%;
    margin: 0 auto; /* Set to just 0 if you don't want these centered inside the tiles */
  } 

  p {
    text-align: center;
    /* Or whatever font settings and spacing as desired */
  }
}
```

Note that this makes use of Sass nesting to create `.portfolio li`, etc. styles.